### DISCUSSION TITLE
Initial position on specific node

### DISCUSSION THREAD

**Discussion by Karpengold** [xyflow/xyflow#2755] [Q&A] [answered]
I have the root node (x and y coords). Is it possible to have it in the top center position of the initial view?

**Comment by moklick** [comment, marked as answer]
That's the case when you set the node position to `{ x: 0, y: 0 }`  https://codesandbox.io/s/dreamy-turing-6l7ryw?file=/src/initial-elements.js:79-108

**Reply by Karpengold** [reply]
And as I understood if positions are defined somewhere else it is not possible to set reactflow on which node to focus on initial view, right?
